### PR TITLE
SD-8319: fix an initialization issue with canvas width

### PR DESCRIPTION
**Pull request by qinyux2021**
SD-8319: fix an initialization issue with canvas width. Initialize the width of the canvas from supplied width when it is loaded first time.

### CODE REVIEW

File: lib/js/reactGUI/LiterallyCanvas.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ var LiterallyCanvas = createReactClass({
 
     var style = {};
     if (imageSize.height) style.height = imageSize.height;
+    if (imageSize.width) style.width = imageSize.width;
```
Comment (gschmidt-msft) [inline lib/js/reactGUI/LiterallyCanvas.js:101]: You able to add a quick inline comment suggesting why we need this change? Might be easier than looking through git history.

Comment (qinyux2021) [reply lib/js/reactGUI/LiterallyCanvas.js:101]: Done

Comment (gschmidt-msft) [reply lib/js/reactGUI/LiterallyCanvas.js:101]: Thanks, but I meant explain why we're needing to do this, as opposed to just stating the obvious.

I thought you mentioned you had spent quite a while debugging this, and then discovered it was setting width in this library. Was just trying to highlight the work you did to uncover this issue.

**Review by gschmidt-msft — APPROVED** [review]

**Comment by qinyux2021** [comment]
The issue is quite simple even though we are not sure why original author intentionally left width out. I guess is that non-React component would not have this initialization issue. So not too much to explain as comments in the code.

From: Graham Schmidt ***@***.***>
Date: Friday, May 27, 2022 at 10:26 AM
To: ComplyMD/literallycanvas ***@***.***>
Cc: Xu, Qinyu ***@***.***>, Author ***@***.***>
Subject: [EXTERNAL] Re: [ComplyMD/literallycanvas] SD-8319: fix an initialization issue with canvas width (PR #2)
CAUTION: This Email is from an EXTERNAL source. Ensure you trust this sender before clicking on any links or attachments.
________________________________

@gschmidt-nuance approved this pull request.

________________________________

In lib/js/reactGUI/LiterallyCanvas.js<https://urldefense.com/v3/__https:/github.com/ComplyMD/literallycanvas/pull/2*discussion_r883667918__;Iw!!L7QdHkQ!lPOxJS71pplJlPBxBBv8kiHuFFO8ChaBnENNj4NwX2kbVeO_Wys7254X0cFmvYX_NFUUbaLLKErxfh3_QfgortC_$>:

> @@ -97,6 +97,7 @@ var LiterallyCanvas = createReactClass({



     var style = {};

     if (imageSize.height) style.height = imageSize.height;

+    if (imageSize.width) style.width = imageSize.width;

Thanks, but I meant explain why we're needing to do this, as opposed to just stating the obvious.

I thought you mentioned you had spent quite a while debugging this, and then discovered it was setting width in this library. Was just trying to highlight the work you did to uncover this issue.

**Pull request merged**